package/MDAnalysis/coordinates/MOL2.py

```diff
@@ -197,7 +197,11 @@ def parse_block(self, block):
 
         coords = np.zeros((self.n_atoms, 3), dtype=np.float32)
         for i, a in enumerate(atom_lines):
-            aid, name, x, y, z, atom_type, resid, resname, charge = a.split()
+            try:
+                aid, name, x, y, z, atom_type, resid, resname, charge = a.split()
```

Comment:
I think here (and other pieces with a similar pattern) it would be cleaner to slice (ie `a, b, c, d = a.split()[:4]`) to the desired size, which is benign if the extra field isn't present.